README.md

```diff
@@ -335,6 +335,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**KDE Connect**](https://invent.kde.org/network/kdeconnect-android) <sup>**[[F-Droid](https://f-droid.org/app/org.kde.kdeconnect_tp)]**</sup>
 * [**Zorin Connect**](https://github.com/ZorinOS/zorin-connect-android) <sup>**[[F-Droid](https://f-droid.org/app/com.zorinos.zorin_connect)]**</sup>
 * [**LocalSend**](https://github.com/localsend/localsend) <sup>**[[F-Droid](https://www.f-droid.org/app/org.localsend.localsend_app)]**</sup>
+* [**Sharing**](https://github.com/Ammar64/Sharing) <sup>**[[F-Droid](https://www.f-droid.org/app/com.ammar.sharing)]**</sup>
```

Comment:
Please keep the entries sorted alphabetically.

Reply:
It's good now.